lib/content-services/src/lib/tag/tags-creator/tags-creator.component.html

```diff
@@ -1,54 +1,40 @@
 <div class="adf-tags-creation">
+    <div *ngIf="isNameTagsVisible" class="adf-tag-name-field">
+        <mat-form-field class="adf-tag-search-field" appearance="fill">
+            <input #tagNameInput
+                    matInput 
+                    autocomplete="off"
+                    [formControl]="tagNameControl"
+                    (keyup.enter)="addTag()"
+                    adf-auto-focus
+                    placeholder="{{'TAG.TAGS_CREATOR.INPUT_PLACEHOLDER' | translate}}" 
+            />
+            <mat-error *ngIf="tagNameControl.invalid">{{ tagNameErrorMessageKey | translate }}</mat-error>
+        </mat-form-field >
+    </div>
     <p
        class="adf-no-tags-message"
-       *ngIf="!tags.length && !tagNameControlVisible">
+       *ngIf="!hasTags">
         {{ 'TAG.TAGS_CREATOR.NO_TAGS_CREATED' | translate }}
     </p>
     <div
         class="adf-tags-list"
         [class.adf-tags-list-fixed]="!tagNameControlVisible"
         #tagsList>
-        <p
-            *ngFor="let tag of tags"
-            class="adf-tag adf-label-with-icon-button">
+        <span
+        *ngFor="let tag of tags"
```

Comment:
Looks like indentation here is incorrect

Reply:
corrected it.